pages/admin/request/create-renewal.tsx

```diff
@@ -114,85 +114,85 @@ export default function CreateRenewal() {
   /**
    * Get information about applicant to pre-populate form
    */
-  const [getApplicant] = useLazyQuery<GetApplicantRenewalResponse, GetApplicantRenewalRequest>(
-    GET_APPLICANT_RENEWAL_QUERY,
-    {
-      fetchPolicy: 'network-only',
-      onCompleted: data => {
-        // set personal information card
-        setPersonalInformationCard({
-          id: +data.applicant.id,
-          rcdUserId: data.applicant.rcdUserId || 0,
-          firstName: data.applicant.firstName,
-          lastName: data.applicant.lastName,
-          gender: data.applicant.gender,
-          dateOfBirth: data.applicant.dateOfBirth,
-          email: data.applicant.email,
-          phone: data.applicant.phone,
-          province: data.applicant.province,
-          city: data.applicant.city,
-          addressLine1: data.applicant.addressLine1,
-          addressLine2: data.applicant.addressLine2,
-          status: data.applicant.status,
-          postalCode: data.applicant.postalCode,
-        });
-
-        // set permitHolderInformation
-        setPermitHolderRcdUserID(data.applicant.rcdUserId || undefined);
-        setApplicantID(+data.applicant.id);
-        setPermitHolderInformation({
-          firstName: data.applicant.firstName,
-          lastName: data.applicant.lastName,
-          email: data.applicant.email,
-          phone: data.applicant.phone,
-          addressLine1: data.applicant.addressLine1,
-          addressLine2: data.applicant.addressLine2,
-          city: data.applicant.city,
-          postalCode: data.applicant.postalCode,
-        });
+  const [getApplicant, { loading: userSearchLoading }] = useLazyQuery<
```

Comment:
Let's use consistent naming for this page and `/create-replacement`:
- `getApplicantLoading` for the `getApplicant` query
- `submitRequestLoading` for the submit mutations